api/RTCDtlsTransport.json

```diff
@@ -124,10 +124,15 @@
               "version_added": "72"
             },
             "chrome_android": "mirror",
-            "edge": {
-              "version_added": "15",
-              "alternative_name": "transport"
-            },
+            "edge": [
+              {
+                "version_added": "79"
+              },
+              {
+                "version_added": "15",
+                "alternative_name": "transport"
```

Comment:
Should there be a `version_removed` for this, or is it really still in Edge, but not in Chrome?